esiclient/tests/functional/README.md

```diff
@@ -0,0 +1,16 @@
+# esi Client Functional Tests
+
+These tests are designed to ensure Ironic and Neutron behave as expected when configured with the Ironic policy file given here: [a link](https://github.com/CCI-MOC/esi/blob/master/etc/ironic/policy.yaml.sample).
+
+### Prerequisites
+
+These tests are intended to be ran against a functioning Openstack cloud with Ironic and Neutron services enables and running. It assumes that the ironic.conf file
+is located at `/etc/ironic/ironic.conf`. These tests assume that the `fake-hardware` hardware type is enabled under the `enabled_hardware_types` option under `[DEFAULT]` heading.
```

Comment:
Two incredibly nitpicky things: Openstack should probably be formatted as OpenStack and also it might be useful to mention that you can pass in a custom Ironic cfg file via the `OS_IRONIC_CFG_PATH` env var for testing purposes.